src/pages/myGeotab/guides/concepts.tsx

```diff
@@ -0,0 +1,1265 @@
+        </p>
+        <CodeSample
+        language="javascript"
+        code={`{
+    "method": "ExecuteMultiCall",
+    "params": {
+        "calls": [
+            {
+                "method": "GetCountOf",
+                "params": {
+                    "typeName": "Device"
+                }
+            },
+            {
+                "method": "GetCountOf",
+                "params": {
+                    "typeName": "User"
+                }
+            }
+        ],
+        "credentials": {
+            "database": "demo",
+            "sessionId": "xxx",
+            "userName": "bob@geotab.com"
+        }
+    }
+}`
+}></CodeSample>
+        <p>Response:</p>
+        <CodeSample
+        language="javascript"
+        code={`{
+    "result": [
+        2340,
+        2022
+    ],
+    "jsonrpc": "2.0"
+}`
+}></CodeSample>
+    </div>
+);
+
+const MultiCallErrors: ReactNode = (
+    <div className="paragraph" id="multiCallErrors">
+        <p>
+            In a MultiCall, each request is run on the server synchronously. If one fails, the error results are returned immediately and <b>unreached calls are not run</b>. The error results includes the index of the call in the array that the exception occurred. 
+        </p>
+        <p>
+            To illustrate, let’s assume an array of calls (api.multicall([call-a, call-b, call-c])) where call-b is formatted incorrectly.
+        </p>
+        <CodeSample
+        language="javascript"
+        code={`var calls = [
+    call-a, // ran successfully
+    call-b, // error occurred, throw and return error
+    call-c  // never ran
+]`
+}></CodeSample>
+        <p>
+            Below is an example of the error result. The <code className="small-code-sample">requestIndex</code> property contains the index of the call that failed.
+        </p>
+        <CodeSample
+        language="javascript"
+        code={`results = {
+    "error": {
+        "message": "The method 'Foobar' could not be found. Verify the method name and ensure all method parameters are included.",
+        "code": -32601,
+        "data": {
+            "id":"2901ac83-0d7f-41a1-9cca-fd4a68e77ae7",
+            "type":"MissingMethodException",
+            "requestIndex": 1
+        }
+    },
+    "jsonrpc":"2.0"
+}`
+}></CodeSample>
+        <p>Alternatively, a successful MultiCall would look similar to:</p>
+        <CodeSample
+        language="javascript"
+        code={`calls = [
+    call-a, // ran successfully
+    call-b, // ran successfully
+    call-c  // ran successfully
+]
+results = {
+    "results": [
+        [...],
+        [...],
+        [...]
+    ]
+}`
+}></CodeSample>
```

Comment:
```suggestion
}/>
```
This component can be self-closing since there are no children.